board/common/post-build.sh

```diff
@@ -80,3 +82,10 @@ grep -qsE '^/bin/true$$' "$TARGET_DIR/etc/shells" \
         || echo "/bin/true" >> "$TARGET_DIR/etc/shells"
 grep -qsE '^/bin/false$$' "$TARGET_DIR/etc/shells" \
         || echo "/bin/false" >> "$TARGET_DIR/etc/shells"
+
+
+if [ -n "$BR2_PACKAGE_NGINX" ]; then
+    cp "$common/netbrowse.conf" "$TARGET_DIR/etc/nginx/"
+    cp "$common/nginx.conf" "$TARGET_DIR/etc/nginx/"
+    cp "$common/ssl.conf" "$TARGET_DIR/etc/nginx/"
+fi
```

Comment:
Any particular reason for not having this in `rootfs/etc/nginx`?

Reply:
Nope, just cruft from the initial prototype. Fixing ...